### DISCUSSION TITLE
Fading Colour change in datagrid on positive/negative value change in a cell

### DISCUSSION THREAD

**Discussion by dbrshell** [perspective-dev/perspective#2645] [Q&A]
Hey Guys,
is it possible to configure the UI in such a way that the datgrid shows a background colour change when an updated value overrides an existing one ? If the updated value is bigger then I would expect a green cell background colour, if the updated value is smaler, then lI would expect a red background colour. Back background colour should fade back to transparent after a few seconds.

Thanks for your feedback !

**Comment by texodus** [comment]
https://github.com/finos/perspective/assets/60666/cbd145af-ddc9-4567-97d8-01fdf8553f0c

**Reply by dbrshell** [reply]
that is fantastic thanks
